ci/scripts/detect-jdk-updates.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -e
```

Comment:
This results in a silent failure if the `curl` command fails. I wonder if something like this might be better:

```bash
report_error() {
	echo "Error $1 on line $2"
	exit 1;
}

trap 'report_error $? $LINENO' ERR
```

Reply:
I like it. Added it.